File: 백준/Bronze/28224. Final Price/Final Price.ts
import * as readline from 'readline';

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let total = 0;
let count = 0;

rl.question('', (n: string) => {
    const num = parseInt(n);
    
    rl.on('line', (input: string) => {
        total += parseInt(input);
        count++;
        
        if (count === num) {
            console.log(total);
            rl.close();
        }
    });
});

File: 백준/Bronze/28224. Final Price/Final Price.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let n: i32 = input.trim().parse().expect("Input was not a number");
    
    let mut total = 0;
    
    for _ in 0..n {
        input.clear();
        io::stdin().read_line(&mut input).expect("Failed to read line");
        let a: i32 = input.trim().parse().expect("Input was not a number");
        total += a;
    }
    
    println!("{}", total);
}